resources/views/wordsLearned.blade.php

```diff
@@ -0,0 +1,51 @@
+@extends ('layouts.app')
+
+@section('title', 'Dashboard | words leaned')
+
+@section('content')
+<div class="container">
+    <div class="row">
+        <div class="col-md-5 col-sm-12">
+            <h4 class="mb-3">Dashboard</h4>
+            <div class="row">
+                <div class="col-md-6 mb-3">
+                    @if ($user->avatar != NULL)
+                        <img src="/storage/profile_images/{{ $user->id }}.jpg" class="w-100" style="objcet-fit:cover;">
```

Comment:
it will not show the image i uploaded
<a href="https://gyazo.com/fb36fe3e77dc5d7d546607ffd4b11b69"><img src="https://i.gyazo.com/fb36fe3e77dc5d7d546607ffd4b11b69.png" alt="Image from Gyazo" width="465"/></a>